src/StructureFactors.jl

```diff
@@ -606,3 +606,72 @@ function accum_dipole_factor_wbasis!(res, S, lattice::Lattice)
         end
     end
 end
+
+
+
+""" FormFactor(q::Array{Float64,1}, elem::String)
```

Comment:
Doc comment in triple quotes are space sensitive. Following Julia stdlib examples, I would suggest changes along these lines
```
"""    FormFactor(elem::String, q )

Compute the form factor for the wave vector `q` measured in inverse Angstroms, and associated with a magnetic ion of species `elem`.
[...]
"""
```